From c9db9953296709ba3c86a7b0471fcd60469225ed Mon Sep 17 00:00:00 2001
From: Jim Ramsay <jim_ramsay@dell.com>
Date: Mon, 1 Nov 2010 08:43:17 -0400
Subject: [PATCH 2/3] Use gkrellm_gkd_string_width

This wrapper is designed to transition from GdkFont to
PangoFontDescription.  Without it, there is a warning and a probable
runtime crash, as we are feeding the wrong structure into
'gdk_string_width'.
--- a/mailwatch.c
+++ b/mailwatch.c
@@ -608,7 +608,7 @@ update_plugin(void) {
 
       p->statstext->x_off =
         gkrellm_chart_width() -
-        gdk_string_width(p->panel->textstyle->font,
+        gkrellm_gdk_string_width(p->panel->textstyle->font,
                          buf) -
         2 * gkrellm_get_style_margins(p->panel->style)->left;
 
-- 
1.7.3.1

